### PR TITLE
Qa 4576 q 4 2022 automation bugs and script fixes

### PR DESCRIPTION
**Pull request by kbo001**
## Summary
Updated steps to fix export downloads and rule deletions

## Description

Updated steps to fix export downloads and rule deletions

### Link to Jira Ticket (if applicable)
[Jira link](https://dimagi-dev.atlassian.net/browse/QA-4576)

## QA Checklist

- [X] Label(s) and Assignee added
- [X] PR sent for review
- [ ] Documentation (if applicable) added/updated

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -185,6 +185,10 @@ USH_Apps/NY_CICT/Tests/output.xml
 /.pabotsuitenames
 /pabot_results/*
 
+#Ignorig Formplayer Fils
```
Comment (kiran-tripathy) [inline .gitignore:188]: typo: Fils instead of Files

File: Formplayer/README.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+
+ -   Copy `settings-sample.cfg` to `settings.cfg` and populate `settings.cfg` for
+the environment you want to test.
+- Run tests using pytest command like:
+
+```sh
+
+# To execute all the test cases 
+pytest -v --rootdir= HQSmokeTests/testCases
```
Comment (kiran-tripathy) [inline Formplayer/README.md:37]: Root directory needs to change

File: Formplayer/email_pass.html [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<html>
+<p>Hello!</p>
+<p>&nbsp; &nbsp; &nbsp; &nbsp; All the Commcare HQ Smoke Tests PASSED successfully.</p>
```
Comment (kiran-tripathy) [inline Formplayer/email_pass.html:3]: HQ Smoke tests in the body. Also, can't we make these templates generic?

File: Formplayer/requires.txt [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+selenium>=3.141.0
+webdriver-manager>=3.2.2
+openpyxl>=3.0.6
+matplotlib >= 3.3.4
+pytest-rerunfailures>=10.2
+pytest-xdist>=2.4.0
+pytest-xdist[psutil]
+Appium-Python-Client == 0.52
+pyotp >=2.6.0
+pytest-order
```
Comment (kiran-tripathy) [inline Formplayer/requires.txt:16]: Do we need all these ? Like Appium, for instance?

File: Formplayer/testCases/conftest.py [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+    settings.read(path)
+    ## updates the url with the project domain while testing in local
+    if settings["default"]["url"] == "https://www.commcarehq.org/":
+        settings["default"]["url"] = f"{settings['default']['url']}a/qa-automation-prod"
+    else:
+        settings["default"]["url"] = f"{settings['default']['url']}a/qa-automation"
+    return settings["default"]
+
+
+@pytest.fixture(scope="module", autouse=True)
```
Comment (kiran-tripathy) [inline Formplayer/testCases/conftest.py:81]: These fixtures were moved to common utilities. https://github.com/dimagi/dimagi-qa/blob/master/common_utilities/fixtures.py

**Comment by kbo001** [comment]
@kiran-tripathy sorry for this misunderstanding. I only wanted to push the last commit but all the changes from formplayer are also pushed. I will be deleting this branch and create a new one.

File: Formplayer/utilities/email_pytest_report.py [hunk 6]
```diff
@@ -0,0 +1,177 @@
+"""
+Qxf2 Services: Utility script to send pytest test reports email
```
Comment (kiran-tripathy) [inline Formplayer/utilities/email_pytest_report.py:2]: is this file required?
